lib/optimizely/notification_center.rb

```diff
@@ -87,6 +87,12 @@ def remove_notification_listener(notification_id)
     end
 
     def clear_notifications(notification_type)
+      # Deprecated
```

Comment:
Find out deprecated meta-tag for docs so that this methods shows up as deprecated in auto generated docs.